src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx

```diff
@@ -66,4 +70,22 @@ function ProductTemplate({ header, children, contentWidth = 6 }: ProductTemplate
   );
 }
 
-export { ProductTemplate };
+type PreLabelProps = {
+  title: string;
+  subtitle: string;
+};
+
+const PreLabel = ({ title, subtitle }: PreLabelProps) => {
```

Comment:
I'm not sure if we should move this to a molecule, probably should do.

Reply:
whats prelabel visually? text that goes on top of progress?